src/coreclr/dlls/mscorrc/mscorrc.rc

```diff
@@ -738,8 +738,8 @@ BEGIN
         IDS_E_PROF_NOT_ATTACHABLE               "Loading profiler failed.  The profiler COM object was instantiated, but the profiler does not support attaching to a live process.  The profiler must be loaded at application startup by using a launcher program included with the profiler (if any) or by setting the COR_ENABLE_PROFILING and COR_PROFILER environment variables before launching the application to be profiled.  Profiler CLSID: '%s'"
         IDS_E_PROF_UNHANDLED_EXCEPTION_ON_LOAD  "Loading profiler failed.  There was an unhandled exception while trying to instantiate the profiler COM object.  Please ensure the CLSID is associated with a valid profiler designed to work with this version of the runtime.  Profiler CLSID: '%s'."
         IDS_PROF_ATTACH_REQUEST_RECEIVED        "The CLR received a request to attach a profiler. Profiler CLSID: '%s'."
-        IDS_PROF_DETACH_INITIATED               "The profiler currently in use has requested to be detached from the process.  The CLR has disabled communication with the profiler and will unload the profiler when it is safe to do so."
```

Comment:
They are printed to the debugger and without a newline it would get printed all on one line. I added newlines to all the profiler ones so they are consistent